packages/flutter_test/lib/src/window.dart

```diff
@@ -606,62 +621,120 @@ class FakeAccessibilityFeatures implements ui.AccessibilityFeatures {
   }
 }
 
+/// Used to fake insets and padding for [TestFlutterView]s.
+///
+/// See also:
+///
+///   * [TestFlutterView.padding], [TestFlutterView.systemGestureInsets],
+///     [TestFlutterView.viewInsets], and [TestFlutterView.viewPadding] for test
+///     properties that make use of [FakeViewPadding].
+class FakeViewPadding implements ViewPadding {
```

Comment:
Let's mark this one as `@immutable` (can't do this for the original one in the engine because the engine cannot depend on package:meta)

Reply:
Done!